api/types/externalcloudaudit/externalcloudaudit.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externalcloudaudit
+
+import (
+	"time"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/api/types/header"
+	"github.com/gravitational/teleport/api/types/header/convert/legacy"
+	"github.com/gravitational/teleport/api/utils"
+)
+
+// ExternalCloudAudit
```

Comment:
Reduntant comment. However, instead of deleting it, I would reference the proto message for documenting the concept itself, and perhaps explain the role of this structure in the overall stack. Something along the lines of "An internal representation of ExternalCloudAudit, as defined in [somewhere], for using in the [some service] API and its clients".